### DISCUSSION TITLE
📢 qiankun logo 发布啦

### DISCUSSION THREAD

**Discussion by kuitos** [umijs/qiankun#1304] [Announcements]
震惊！微前端开源社区星星最多的 qiankun 框架，竟然连一个自己的 logo 都没有！

今天，我们在 [dumi](https://github.com/umijs/dumi) 作者、qiankun「御用」设计师 @PeachScript 的帮助下，终于迎来了 qiankun 第一个版本的 logo 啦！大家鼓掌👏👏👏

<img src="https://user-images.githubusercontent.com/5206843/110293989-391b3c00-802a-11eb-81f4-382ee76bf7a5.png" alt="qiankun logo">

关于 logo 的设计灵感，且看 @PeachScript 的官方解答：
> 在形体上，qiankun 的 LOGO 基于方形及圆形进行衍生。乾坤本为《易》中的两个卦象，乾卦 ☰、坤卦 ☷，加之后人常用乾坤指代天地，所以 LOGO 中将圆形衍生、比作天际的同时将乾卦暗含其中；而方形则变幻、寓作大地的同时将坤卦藏于中间。
> 
> 在细节上，『方圆几里』的层层嵌套，体现了 qiankun 作为微前端框架，将任意技术栈的子应用环环相扣、紧密相连、共生万象的核心价值；而『大地』上坤卦形成的负空间，犹如一支正在冲破云霄的火箭，背后是无垠的黄色土地，前方是未知的广袤星空，qiankun 从不畏惧。

大家对这个 logo 有什么不一样的想法或建议，欢迎在下面回帖留言👇

## 搭车招聘，诚招天下英雄

如果您对微前端感兴趣，请发简历到 [youzhi.lk@antfin.com](mailto:youzhi.lk@antfin.com)，我们非常期望有机会能与您共事，探索出微前端下更多的场景可能性。

如果您对微前端没什么兴趣，不要紧，只需要您对 antd、AntV、dva、umi、eggjs、ahooks 中任一类别的领域感兴趣，您也可以发简历到 [youzhi.lk@antfin.com](mailto:youzhi.lk@antfin.com) 来与我们共商大事。

### 团队介绍

- 大部门：蚂蚁集团体验技术部
- 部门主管：玉伯
- 部门开源作品：antd、AntV、dva、umi、eggjs、qiankun、ahooks 等
- 部门商业化产品：语雀
- 直属部门：平台前端技术部 - 部门主管：偏右（antd 负责人），部门介绍：https://www.yuque.com/afx/about/skyline
- 要求：基本没要求，p5~p8 我们都需要，只要您想来试试就可以投简历到 [youzhi.lk@antfin.com](mailto:youzhi.lk@antfin.com)，HC 非常多！！
- base 地：杭州、上海、成都 三地任选

**Comment by hql7** [comment]
鼓掌，撒花，第一眼感觉像火箭，寓意qiankun要火出宇宙啊



在 2021-03-08 16:15:05，"Kuitos" <notifications@github.com> 写道：

震惊！微前端开源社区星星最多的 qiankun 框架，竟然连一个自己的 logo 都没有！

今天，我们在 dumi 作者、qiankun「御用」设计师 @PeachScript 的帮助下，终于迎来了 qiankun 第一个版本的 logo 啦！大家鼓掌👏👏👏

关于 logo 的设计灵感，且看 @PeachScript 的官方解答：

在形体上，qiankun 的 LOGO 基于方形及圆形进行衍生。乾坤本为《易》中的两个卦象，乾卦 ☰、坤卦 ☷，加之后人常用乾坤指代天地，所以 LOGO 中将圆形衍生、比作天际的同时将乾卦暗含其中；而方形则变幻、寓作大地的同时将坤卦藏于中间。

在细节上，『方圆几里』的层层嵌套，体现了 qiankun 作为微前端框架，将任意技术栈的子应用环环相扣、紧密相连、共生万象的核心价值；而『大地』上坤卦形成的负空间，犹如一支正在冲破云霄的火箭，背后是无垠的黄色土地，前方是未知的广袤星空，qiankun 从不畏惧。

**Comment by PeachScript** [comment]
给 qiankun 疯狂打 call！！！

**Comment by taosin** [comment]
日出东方

**Comment by anLl** [comment]
牛转乾坤~

**Comment by nickolboy** [comment]
基站已搭建完毕，开启全网广播 :P

**Comment by gongshun** [comment]
只看logo，很难想到这是qiankun 😂
